src/js/index.js

```diff
@@ -21,9 +21,51 @@ const allowDifficultySelect = shouldAllow => sortingInput.querySelectorAll('.dif
 		node.disabled = !shouldAllow;
 	});
 
+const parameters = {
+	tags: {
+		default: '',
+		getValue: () => selectr.getValue().join(' '),
+		setValue: value => selectr.setValue(value.split(' '))
+	},
+	difficulty: {
+		default: 'alldifficulties',
+		getValue: () => filterInput.value,
+		setValue: value => {
+			filterInput.value = ['easy', 'medium', 'difficult'].includes(value) ? value : 'alldifficulties';
+		}
+	},
+	sort: {
+		default: 'ALPHABETICAL_ASCENDING',
```

Comment:
we should split this into `order` and `sortMethod`

i.e. `?sortMethod=alphabetical&order=asc`